perun-beans/src/main/java/cz/metacentrum/perun/core/api/DBVersion.java

```diff
@@ -0,0 +1,66 @@
+package cz.metacentrum.perun.core.api;
+
+import java.util.List;
+
+/**
+ * Object for representation of database version.
+ *
+ * @author Simona Kruppova 410315
+ * @author Oliver Mrazik 410035
+ */
+public class DBVersion {
+	String version;
+	List<String> commands;
```

Comment:
It is good practice to use private before variables in objects. For purpose of setting and getting them there are public methods.